### PR TITLE
Replace Special keys with icons

### PR DESCRIPTION
**Pull request by chrisgrieser**
Make the hotkeys look a little bit nicer. :)

### CODE REVIEW

File: utils.ts [hunk 1]
```diff
@@ -37,7 +47,12 @@ export function generateHotKeyText(hotkey: Hotkey): string {
         hotKeyStrings.push(MODIFIER_ICONS[mod])
     }
 
-    hotKeyStrings.push(hotkey.key.toUpperCase())
+    var key = hotkey.key.toUpperCase();
+    SPECIAL_KEYS.forEach (special_key =>{
+        if (special_key[0] === key) key = special_key[1];
+    });
+
+    hotKeyStrings.push(key)
```
Comment (AlexBieg) [inline utils.ts:55]: With the dict above we can also clean this up a little. Also it tends to be better to use `let` over `var`. `var` gets hoisted which has caused my grief in the past. In our case here though we can just use const.

```suggestion
    const key = hotkey.key.toUpperCase();
    hotKeyStrings.push(SPECIAL_KEYS[key] || key)
```

File: utils.ts [hunk 2]
```diff
@@ -37,7 +47,12 @@ export function generateHotKeyText(hotkey: Hotkey): string {
...
 
     return hotKeyStrings.join(' ')
-}
\ No newline at end of file
+}
```
Comment (AlexBieg) [inline utils.ts:58]: 🔥

**Review by AlexBieg — CHANGES_REQUESTED** [review]
Thanks so much! I really appreciate the help. This looks way better in the palette. The functionality looks super solid to me. Just a couple of minor code clean up requests 😄

**Comment by chrisgrieser** [comment]
yep, accepted your commits, are all fine with me :)

**Review by AlexBieg — APPROVED** [review]
I forgot one thing in my suggestions so I just added that. Thanks for making those changes! Merging it now 😄

**Pull request merged**